app/experimenter/experiments/constants/nimbus.py

```diff
@@ -65,6 +65,18 @@ class NimbusTargetingConfig:
     ),
 )
 
+TARGETING_HOMEPAGE_GOOGLE = NimbusTargetingConfig(
+    name="Homepage set to google.com",
+    slug="homepage_google_dot_com",
+    description="Users with their Homepage set to google.com",
+    targeting=(
```

Comment:
I think you probably want to add `TARGETING_US_ONLY.targeting` in here

Reply:
Done